### PR TITLE
docs: add configuration guide for extensions and receivers

### PR DESCRIPTION
**Pull request by kkujawa-sumo**

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -1,40 +1,220 @@
...
+    scrapers:
+      load:
+
+# Define your processors...
+processors:
+
+exporters:
+  sumologic:
+    auth:
+      authenticator: sumologic
```
Comment (sumo-drosiek) [inline docs/Configuration.md:38]: Can we add example with `sumologic/1`?
`sumologic` is default auth in exporter now

File: examples/config_cpu_load_metrics.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    scrapers:
+      load:
+
+# Define your processors...
+processors:
+
+exporters:
+  sumologic:
+    auth:
+      authenticator: sumologic
```
Comment (sumo-drosiek) [inline examples/config_cpu_load_metrics.yaml:20]: We can skip auth as we set it by default :)

File: docs/Configuration.md (hunk 1 above)
Comment (pmalek-sumo) [reply docs/Configuration.md:38]: Separate example explaining `sumologic/1`

File: docs/Configuration.md [hunk 3]
```diff
@@ -1,40 +1,220 @@
...
+The basic configuration for Telegraf Receiver has following format:
+
+```yaml
+receivers:
+  telegraf:
+    separate_field: false
+    agent_config: |
+      [agent]
+        interval = "2s"
+        flush_interval = "3s"
```
Comment (pmalek-sumo) [inline docs/Configuration.md:169]: I'm not 100% sure about this but I'd set those to the same value.

File: examples/config_cpu_load_metrics.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+service:
+  extensions: [sumologic]
+  pipelines:
+    metrics:
+      receivers: [hostmetrics]
+      processors: []
```
Comment (pmalek-sumo) [inline examples/config_cpu_load_metrics.yaml:27]: Recently I've discovered that this is not necessary

File: examples/config_cpu_load_metrics.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+# Define your receivers...
+receivers:
+  hostmetrics:
+    collection_interval: 30s
+    scrapers:
+      load:
+
+# Define your processors...
+processors:
```
Comment (pmalek-sumo) [inline examples/config_cpu_load_metrics.yaml:15]: ... as well as this

File: docs/Configuration.md (hunk 1 above)
Comment (sumo-drosiek) [reply docs/Configuration.md:38]: After looking in code we don't recognize multiple extensions, so no need to document it
@pmalek-sumo is that correct statement?

File: examples/config_cpu_load_metrics.yaml (hunk 4 above)
Comment (kkujawa-sumo) [reply examples/config_cpu_load_metrics.yaml]: removed

File: examples/config_cpu_load_metrics.yaml (hunk 5 above)
Comment (kkujawa-sumo) [reply examples/config_cpu_load_metrics.yaml]: removed

File: docs/Configuration.md (hunk 3 above)
Comment (kkujawa-sumo) [reply docs/Configuration.md]: set to the same value

File: examples/config_cpu_load_metrics.yaml (hunk 2 above)
Comment (kkujawa-sumo) [reply examples/config_cpu_load_metrics.yaml]: I removed this from example.

File: docs/Configuration.md (hunk 1 above)
Comment (pmalek-sumo) [reply docs/Configuration.md]: Yup, for now we only take the first one found. https://github.com/SumoLogic/sumologic-otel-collector/blob/d4157051abcfb6a82baaf587d8a813afe3e04a45/pkg/exporter/sumologicexporter/exporter.go#L379-L386

We should change that to account for more extensions being able to be defined, that would (as far as I can tell for now) require exposing Component ID from `config.ExtensionSettings`

Comment (kkujawa-sumo) [reply docs/Configuration.md]: @sumo-drosiek @pmalek-sumo I propose to change documentation when we add support for multiple authenticators in the config.

File: docs/Configuration.md [hunk 6]
```diff
@@ -1,40 +1,213 @@
...
-[filelogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.27.0/receiver/filelogreceiver
-[syslogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.27.0/receiver/syslogreceiver
+[otlp]: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md
 [otlpreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector/tree/v0.27.0/receiver/otlpreceiver
-[telegrafreceiver_readme]: ../pkg/receiver/telegrafreceiver
\ No newline at end of file
+
+### Receivers from OpenTelemetry Collector
+
+Sumo Logic OT Distro has inbuilt receivers from [OpenTelemetry Collector](https://github.com/SumoLogic/opentelemetry-collector) and they can be used in configuration for this distribution.
```
Comment (pmalek-sumo) [inline docs/Configuration.md:192]:
```suggestion
Sumo Logic OT Distro has builtin receivers from 
[OpenTelemetry Collector](https://github.com/SumoLogic/opentelemetry-collector) and they can be used
in configuration for this distribution.
```

Or perhaps let's move the link out of the paragraph and then restructure this, hm?

File: docs/Configuration.md [hunk 7]
```diff
@@ -1,40 +1,213 @@
...
+```yaml
+receivers:
+  hostmetrics:
+    collection_interval: 30s
+    scrapers:
+      load:
+```
+
+for details please see documentation for appropriate receiver.
+Receivers along with documentation can be found [here][opentelemetry-collector-receivers].
```
Comment (pmalek-sumo) [inline docs/Configuration.md:205]:
```suggestion
for details please see documentation for appropriate receiver.

Receivers along with documentation can be found [here][opentelemetry-collector-receivers].
```

**Review by pmalek-sumo — COMMENTED** [review]
Generally ok with some minor nit picks

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**